common/app/routes/Challenges/views/classic/classic.less

```diff
@@ -32,6 +32,9 @@
     }
     .cm-keyword, .cm-attribute {
       color:#9BBBDC;
+    }    
+    .CodeMirror-cursor {
```

Comment:
We need to be more selective here, else `codemirror` styles will win the cascade. 

Use this instead :+1: 
```css
.CodeMirror-cursor {
      border-left-color: @night-text-color;
    }
```